lib/opensusebasetest.pm

```diff
@@ -1000,7 +985,6 @@ sub wait_boot_past_bootloader {
     handle_emergency_if_needed;
 
     handle_broken_autologin_boo1102563()          if match_has_tag('displaymanager');
-    handle_additional_polkit_windows_bsc1157928() if match_has_tag('authentication-required-user-settings');
```

Comment:
I've checked if this issue still occurs on first_boot and it does: https://openqa.suse.de/tests/4795895#step/first_boot/8 therefore we cannot simply remove this workaround, but we need either to reopen the bug or update it with new bugref.

Reply:
Ouch. I think I will just leave this workaround then, and re-open the bug as it's really still the same.

Reply:
Second thought, AFAIS it happens only on s390 so we could change just the bugref and name of subroutine to point to the new one.

Reply:
Yep, that's what I proposed in the first comment. Both options are fine. Fresh bugref is better as it's WIP and the discussions are happening there and not in the old one.

Reply:
So I did that